src/utils/wave/getWaveLeaderBoard.ts

```diff
@@ -0,0 +1,91 @@
+import { polygon } from 'viem/chains'
+import { formatUnits, parseUnits, type Address } from 'viem'
+
+import { getApiUrl, type ChainId } from '../../config'
+import { type WaveStatsResponse } from './getWaveStats'
+import { type WaveLevelData } from './getWaveLevels'
+
+
+type LeaderBoardTotalApiItem = WaveStatsResponse & {
+  position: number
+}
+
+type LeaderBoardPeriodApiItem = {
+  position: number
+  address: Address
+  points: string
+  wavePeriodId: number
+  sharePercent: string
+  expectedPositionMultiplier: string
+}
+
+export type WaveLeaderBoardItem = {
+  position: number
+  address: Address
+  points: string
+  bonusPoints: string | null
+  totalMultipliedPoints: string
+  bonusMultiplier: number | null
+  level: number | null
+  levelDescription: WaveLevelData | null
+}
+
+type Props = {
+  waveId: number
```

Comment:
`waveId` may be `number | 'active'`, and we propose to use 'active' for all clients to show up-to-date data.
Maybe make it optional with `'active'` by default (everywhere)?

Reply:
66a69e9affed5b8a746de1fc7504c36a9a395b66